resources/views/newDesign/socialModule/_index.blade.php

```diff
@@ -1,12 +1,12 @@
 @include('newDesign.socialModule.facebook.set',[
-    'app_name'      => env('APP_NAME'),
```

Comment:
Why you change that line?

Reply:
![screen](https://user-images.githubusercontent.com/32058849/36735959-0e509eea-1be0-11e8-9160-390de411ad9c.jpeg)
There are no APP_NAME in env file

Reply:
I can add